src/components/Header/components/NavAvatarRightCorner/index.jsx

```diff
@@ -20,7 +19,7 @@ const useStyles = makeStyles({
       boxShadow: '0px 1px 2px 0px rgba(0, 0, 0, 0.5)',
     },
     '&_placeholder': {
-      backgroundColor: gordonColors.primary.cyan,
+      className: 'styles.gordon_header_placeholder',
```

Comment:
Gideon and Andrew actually already worked on this, since it's using a deprecated MUI package mui/styles...  Can you revert your changes to this file, and the associated CSS, since they already replaced the whole thing.